tests/10apidoc/30room-create.pl

```diff
@@ -287,8 +288,10 @@ =head2 matrix_create_room
 commandline.
 
 The resultant future completes with two values: the room_id from the
-/createRoom response; the room_alias from the /createRoom response (which is
-non-standard and its use is deprecated).
+/createRoom response; and the room_alias. room_alias will be loaded from 
+/createRoom (if it exists, which is non-standard and its use is deprecated) or
+it will find the alias by loading room's aliases, in case that it's missing
+from /createRoom.
```

Comment:
this is no longer correct.

Reply:
Reworded here: https://github.com/matrix-org/sytest/pull/1216/commits/05aa8459beee07fe803b32420e8aa87b6bc04fba